phpunit/class-wp-rest-navigation-controller-test.php

```diff
@@ -0,0 +1,290 @@
+<?php
+/**
+ * Unit tests covering WP_REST_Navigation_Controller functionality.
+ *
+ * @package WordPress
+ * @subpackage REST API
+ */
+
+/**
+ * @group restapi
+ * @group navigation
+ */
+class WP_REST_Navigation_Controller_Test extends WP_Test_REST_Controller_Testcase {
```

Comment:
Currently these tests assert on the endpoint itself. Perhaps however it would be better to test the `WP_Navigation_Gutenberg` class directly as that code is also consumed by the Navigation block render callback?